hazelcast/src/main/java/com/hazelcast/internal/crdt/AbstractCRDTReplicationOperation.java

```diff
@@ -78,21 +76,11 @@ public int getFactoryId() {
 
     @Override
     protected void writeInternal(ObjectDataOutput out) throws IOException {
-        out.writeInt(replicationData.size());
-        for (Entry<String, T> entry : replicationData.entrySet()) {
-            out.writeString(entry.getKey());
-            out.writeObject(entry.getValue());
-        }
+        SerializationUtil.writeMapStringKey(replicationData, out);
     }
 
     @Override
     protected void readInternal(ObjectDataInput in) throws IOException {
-        final int mapSize = in.readInt();
-        replicationData = createHashMap(mapSize);
-        for (int i = 0; i < mapSize; i++) {
-            final String name = in.readString();
-            final T crdt = in.readObject();
-            replicationData.put(name, crdt);
-        }
+        replicationData = SerializationUtil.readMap(in);
```

Comment:
```suggestion
        replicationData = SerializationUtil.readMapStringKey(in);
```

Reply:
looks like this change was reverted?

Reply:
> looks like this change was reverted?

It's definitely there, GitHub just isn't showing it in the PR diff for some reason?

https://github.com/JackPGreen/hazelcast/blob/remove-replicate-map-serialization-code/hazelcast/src/main/java/com/hazelcast/internal/crdt/AbstractCRDTReplicationOperation.java#L79